### DISCUSSION TITLE
PC freeze with black screen after 3 seconds testing with memtest_vulkan

### DISCUSSION THREAD

**Discussion by Obiwarn** [GpuZelenograd/memtest_vulkan#48] [Card-specific memtest_vulkan error logs]
When I play any game in 4K (lower res is OK) then after one minute the PC crashes with a black screen (computer seems to be still running, but is frozen).

When I start memtest_vulkan I get the same error after 2-3 seconds. 

Does this hint towards some broken VRAM on the GPU?

**Comment by galkinvv** [comment]
Its hard to distinguish "VRAM-caused crash" and "some-other-GPU-proble-caused crash"

memtest-vulkan tries to save errors in log file, sometimes errors are detected "just before crash" and are saved. So, please attach yours memtest-vulkan.log.
(but, this not always the case, sometimes the crash is befor getting any errors, in this situation it typically is not rlated to VRAM)

**Comment by Obiwarn** [comment]
Thank you for the quick response.

The log file is:
```
Tester console logging started at 2024-10-03T19:15:31.913053Z

1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
Tester worker logging started at 2024-10-03T19:15:31.992936Z
Standard 5-minute test of 1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
      1 iteration. Passed  0.0386 seconds  written:    7.0GB 446.9GB/sec        checked:   10.5GB 458.6GB/sec
     26 iteration. Passed  1.0015 seconds  written:  175.0GB 446.4GB/sec        checked:  262.5GB 430.6GB/sec
    156 iteration. Passed  5.0265 seconds  written:  910.0GB 446.5GB/sec        checked: 1365.0GB 456.7GB/sec
Tester console logging started at 2024-10-03T20:20:50.496904Z

1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
Tester worker logging started at 2024-10-03T20:20:50.608372Z
Standard 5-minute test of 1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
      1 iteration. Passed  0.0825 seconds  written:    7.0GB 210.1GB/sec        checked:   10.5GB 213.3GB/sec
     14 iteration. Passed  1.0155 seconds  written:   91.0GB 228.6GB/sec        checked:  136.5GB 221.1GB/sec
Error found. Mode INITIAL_READ, total errors 0x6BA out of 0x38000000 (0.00018328%)
Errors address range: 0xF2790900..=0xF2A67FFF  iteration:20
values range: 0xFFD517B6..=0x0005E46A   FFFFFFFF-like count:0    bit-level stats table:
         0x0 0x1  0x2 0x3| 0x4 0x5  0x6 0x7| 0x8 0x9  0xA 0xB| 0xC 0xD  0xE 0xF
SinglIdx  14   8   15  14|  11  15   21   9|  20  11   21  11|  11  10   16  11
   0x1?   13  13   14  12|  19  10    9  12|  16  13   16  10|  11  13   12  13
TogglCnt     424  512 407| 219  88   49  17|   4   1    1    |                 
1sInValu                 |                 |   6   8   29  49| 104 182  217 237
   0x1?  234 210  157 127|  93  50   15   2|            2    |                 
```

**Reply by galkinvv** [reply]
There is was error logged on iteration 20, but I can't be 100% sure it was caused by GPU<->VRAM connectivity, because it doesn't look like a most typical memory error. It can be something like memory power system spike.

You can try setting VRAM clocks and GPU clocks to smallest possible value in MSI afterburner and look if that extremely minial frequencies would be stable. if yes - start raising them step-by-step to find the actual limit.

**Reply by Obiwarn** [reply]
I clocked both GPU and RAM down by max in Afterburner (-502 Mhz).
Now it seems to get a bit further, but crashes without error.

Problem is: My PC fell down from around 25 cm height (before this everything worked perfectly fine), so there could be many reasons. However, the rest seems to run fine, so I guess I have to return the graphics card? Or is the memtest consuming a lot of power (I could not measure since it crashes so fast)? Could it be a PSU error?



The logs:

1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
Tester worker logging started at 2024-10-04T09:58:18.960940Z
Standard 5-minute test of 1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
      1 iteration. Passed  0.0395 seconds  written:    7.0GB 435.1GB/sec        checked:   10.5GB 448.7GB/sec
     27 iteration. Passed  1.0269 seconds  written:  182.0GB 434.0GB/sec        checked:  273.0GB 449.3GB/sec
    157 iteration. Passed  5.0382 seconds  written:  910.0GB 443.8GB/sec        checked: 1365.0GB 456.8GB/sec


Tester console logging started at 2024-10-04T14:15:38.005155Z

1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
Tester worker logging started at 2024-10-04T14:15:38.085739Z
Standard 5-minute test of 1: Bus=0x2B:00 DevId=0x2786   12GB NVIDIA GeForce RTX 4070
      1 iteration. Passed  0.0421 seconds  written:    7.0GB 413.1GB/sec        checked:   10.5GB 418.2GB/sec
     26 iteration. Passed  1.0390 seconds  written:  175.0GB 415.1GB/sec        checked:  262.5GB 425.2GB/sec
    147 iteration. Passed  5.0289 seconds  written:  847.0GB 414.8GB/sec        checked: 1270.5GB 425.3GB/sec

**Reply by galkinvv** [reply]
If the problem started after a drop - quite high chances are that GPU is damaged. The most reliable way would be checking this GPU in another PC.
